#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/initd-header.patch.dpatch by Harald Jenny <harald@a-little-linux-box.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fdebd
## DP: Author: Harald Jenny <harald@a-little-linux-box.at>
## DP: 
## DP: 
## DP:     programs/setup/setup.in: Populate Default-Start

@DPATCH@

diff --git a/programs/setup/setup.in b/programs/setup/setup.in
index 44d99b1..21b1bdd 100755
--- a/programs/setup/setup.in
+++ b/programs/setup/setup.in
@@ -5,7 +5,7 @@
 # Provides:          ipsec
 # Required-Start:    $network $remote_fs $syslog $named
 # Required-Stop:     $syslog $remote_fs
-# Default-Start:
+# Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
 # Short-Description: Start Openswan IPsec at boot time
 # Description:       Enable automatic key management for IPsec (KLIPS and NETKEY)
